typescript-api/src/dancebox/interfaces/lookup.ts

```diff
@@ -804,7 +804,7 @@ export default {
         },
     },
     /**
-     * Lookup73: pallet_data_preservers::pallet::Event<T>
+     * Lookup72: pallet_data_preservers::pallet::Event<T>
```

Comment:
So the typescript-api hack failed because of this, because we didn't have a "Lookup72" before, and now for whatever reason we do. I guess its related to using an Option<u128> above and a bug in the type generation code, wild stuff